src/routes/reference/mlpclassifier.svx

```diff
@@ -6,4 +6,39 @@ tags:
     - classification
     - neural network
     - machine learning
----
\ No newline at end of file
+    - supervised learning
+    - multi-layer perceptron
+    - mlp
+---
+
+The `MLPClassifier` is an object that can be used to perform _classification_. 'MLP' stands for multi-layer perceptron which is a type of neural network. By providing data as a `FluidDataSet` and labels as a `FluidLabelSet`, the neural network is iteratively trained using _supervised learning_ so that it can predict the most probable label output for a given data point input.
+
+See this [YouTube tutorial]() where we train the `MLPClassifier` to recognise oboe or trombone sounds in real-time.
+
+## A Brief Explanation
+
+When this MLP neural network trains, it takes each data point from the FluidDataSet provided as an input and predicts an output guess. At first these guesses will be quite wrong because the MLP begins in a randomized state. After each of these output guesses it will check what output is desired by checking against the label in the provided FluidLabelSet with the same identifier. The MLPClassifier will then adjust its internal parameters slightly based on how wrong it was so that it can be more accurate the next time it makes a prediction on this data point. After training on each data point thousands of times, these small adjustments enable the neural network to (hopefully!) make accurate predictions on the datapoints in the FluidDataSet, as well as on data points it has never seen before.
+
+## Parameters
```

Comment:
That does make sense. I'm cool with removing the listing. 

If we're going to write some prose to:
`"For example, to explain hidden layers one might refer to the @hidden/neurons/layers attribute and how one would change it"` (@jamesb93)
Then maybe that should just go in the CCE documentation? If it's info that the user needs to know we should give it to them right up front I suppose.

If an explanation paragraph like this does go on the Learn site, then it probably should have a heading titled, 'Hidden Layers', which is kind of just changing the list into a series of headings?

I think I was imagining the 'Reference' being pretty bare bones, kind of just a replication of the help files so that in overviews and tutorials on Learn if we say "FluidBufNMF" the user wouldn't have to refer back to their CCE documentation to know what that is. They can just click on that word and get some info. Or if we say "next, predictLabel on the classifier", they can search the site for "predictLabel" and it shows up.

I see why it might be silly to duplicate the documentation in web form but it also might help with learning the tools.

Reply:
I like the categories and such that they use here: https://p5js.org/learn/

Reply:
this is also good: https://ml5js.org/